docs/content/latest/yugabyte-cloud/cloud-security/cloud-security-features.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Security features of Yugabyte Cloud
+linkTitle: Security features
+description: Security features of Yugabyte Cloud.
+image: /images/section_icons/index/secure.png
+menu:
+  latest:
+    parent: cloud-security
+    identifier: cloud-security-features
+weight: 10
+isTocNested: true
+showAsideToc: true
+---
+
+Yugabyte Cloud is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on public cloud providers such as Google Cloud Platform (GCP) and Amazon Web Services (AWS), with more public cloud provider options coming soon. Yugabyte Cloud consists of management and data planes. The management plane is a centralized management service deployed on GCP; the management plane is responsible for creating and managing customer data planes. The data plane hosts the customer YugabyteDB clusters deployed on public cloud provider infrastructure.
```

Comment:
The diagram has **Control plane**. Is it the same as management plane ? If yes , why two different names ? It is not consistent . If no and control plane is different , then there should be information about it.

Reply:
@aishwarya24  where is this naming convention coming from? I see both terms used interchangeable